tests/testthat/test-input-time-series.R

```diff
@@ -152,3 +152,19 @@ test_that("ART data throws error if dupe annual data or incomplete quarterly", {
     paste0("Quarterly data not provided for all disaggregates or duplicate ",
            "annual data provided for all disaggregates."))
 })
+
+test_that("can get plot type descriptions from key", {
+  ret <- get_plot_type_label_and_description(c("art_total", "art_child"))
+  expect_equal(ret, list(
+    list(
+      id = "art_total",
+      label = "ART count",
+      description = "Number on ART at the end of calendar year"
+    ),
+    list(
+      id = "art_child",
+      label = "ART paediatric",
+      description = "Proportion of total on-ART under age 15"
```

Comment:
```suggestion
      description = "Number of children (<15) on ART at the end of calendar year"
```
Not clear whether this should be "Proportion" or "Number" but test failure is due to mismatch with JSON